README.md

```diff
@@ -392,6 +433,13 @@ This is a known issue in DuckDB. If you are using DBeaver, this means shutting d
 
 Very worst-case, deleting the database file will get you back in action (BUT you will lose all your data).
 
+
+#### GitHub Codespaces and VSCode Remote Container
+
+If you're using a privacy-forward browser such as Firefox and Braze, or a tracking-cookie-blocking extension like UBlock Origin or Privacy Badger, you may see the below error. You can either change your cookie settings, use a browser like Chrome, or just ignore the error because it doesn't affect the demo
```

Comment:
```suggestion
If you're using a privacy-forward browser such as Firefox and Brave, or a tracking-cookie-blocking extension like UBlock Origin or Privacy Badger, you may see the below error. You can either change your cookie settings, use a browser like Chrome, or just ignore the error because it doesn't affect the demo
```